benchmark_runner/common/oc/oc.py

```diff
@@ -508,7 +508,7 @@ def delete_all_resources(self, resources: list = ('pods', 'pvc'), namespace: str
         :return:
         """
         for resource in resources:
-            return self.run(f'oc delete -n {namespace} {resource} --all ')
+            return self.run(f'oc delete -n {namespace} {resource} --all --force')
```

Comment:
`oc delete --force` is best avoided unless you are certain that you know what you're doing.  It can lead to node or cluster instability.  See https://access.redhat.com/solutions/4165791.  If something isn't being deleted, it typically indicates that there's a real underlying problem and it's important to figure out what it is.

Reply:
Fixed

Reply:
Still appears to be in there.

Reply:
Done

Reply:
Still see the `--force`, that should be removed.